### ISSUE TITLE
How to use high-level python API to get base logits of input?

### ISSUE THREAD

**Issue by joshuageddes** [abetlen/llama-cpp-python#527]
Hi,

I am trying to benchmark ppl for llama.cpp using a custom Python evaluation harness I made. However, I'm having difficulty understanding how to use the high level api to get the logits of a prompt input; all API examples include the token sampling step of inference.
